test/e2e/e2e_suite_test.go

```diff
@@ -254,11 +254,50 @@ func initRancherTurtles(clusterProxy framework.ClusterProxy, config *clusterctl.
 	}
 	_, err := chart.Run(map[string]string{
 		"cluster-api-operator.cert-manager.enabled": "true",
-		"cluster-api-operator.infrastructure":       config.GetVariable(capiInfrastructure),
-		"cluster-api-operator.secretName":           "variables",
-		"cluster-api-operator.secretNamespace":      "default",
+		"capi.configSecret.namespace":               "default",
+		"capi.configSecret.name":                    "variables",
 	})
 	Expect(err).ToNot(HaveOccurred())
+
```

Comment:
IMO this test case should be added in the CAPI operator repository (nit)

Reply:
Can you elaborate? what exactly has to be tested in CAPI operator?

Reply:
Is [this](https://github.com/rancher-sandbox/rancher-turtles/pull/109/files#diff-eb068e1dbcbf03afb60148e8974e03075b485f799c46b05643c3c2292c0b2154R1-R41 ) supposed to be installed together with operator by the users? Why testing kubeadm provider here when it could be tested in the CAPI operator repository? Additionally with helm uninstall issue, this blocks other operations within e2e tests, it surely missing manual uninstall part in this specific test case. It is not scalable.

Reply:
I will add a manual uninstall until this is fixed in operator

Reply:
so after some thinking, I don't believe we have to uninstall manually providers in e2e, we tear down the entire management cluster and if we need to debug the test run and skip cleanup it's better to leave the providers running, wdyt?

Reply:
This implementation fixes uninstall process in capi operator - https://github.com/kubernetes-sigs/cluster-api-operator/pull/251 so no need to do some unnessesary cluster teardowns, we just need to merge it. Creating and tearing down management clusters is costly, not scalable and will prolong the e2e test execution.